src/components/LogStream/LogsTableHeader.tsx

```diff
@@ -1,16 +1,21 @@
 /** @jsx jsx */
-import { jsx } from "@emotion/core";
+import { jsx } from "@emotion/react";
 import { Row, Col } from "react-bootstrap";
 
 type Props = {
   columns: { firstCol: number; secondCol: number };
-  border?:boolean;
+  border?: boolean;
 };
 
-const LogsTableHeader = ({ columns: { firstCol, secondCol },border}: Props) => {
+const LogsTableHeader = ({
+  columns: { firstCol, secondCol },
+  border,
+}: Props) => {
   return (
     <div
-      className={`px-4 py-1 bg-light text-monospace ${border?'border-top':''}`}
+      className={`px-4 py-1 bg-light text-monospace ${
+        border ? "border-top" : ""
```

Comment:
Please use emotion for styles that depend on props.

You can find an example here https://github.com/jina-ai/dashboard/pull/143/files#diff-a885c9618e08d0a2b0d10d6db0acac866a3d0b5555e6093305941d41e04d39c6R51

Reply:
Thanks for linking the example, will add this

Reply:
This is actually not related to the changes in the current PR. this was simply a formatting change. I think we should tackle this in a broader style refactoring task as there are a lot of these instances in this file as well as many others.